src/main/java/org/spongepowered/api/resourcepack/ResourcePackFactory.java

```diff
@@ -28,7 +28,7 @@
 import java.net.URL;
 
 /**
- * A factory for creating {@link ResourcePack}s.
+ * A provider for creating {@link ResourcePack}s.
```

Comment:
Just a bit. I'll go through and revise javadoc after the core content and
concept are accepted or rejected.